code/game/objects/items/weapons/storage/blackshield_kits.dm

```diff
@@ -0,0 +1,148 @@
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/cell/medium(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/greasegun
+	name = "\improper Grease Gun Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/automatic/greasegun(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/ammo_magazine/smg_35(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+/obj/item/storage/box/bs_kit/grizzly
+	name = "\improper Grizzly Primary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/shotgun/pump(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/ammo_casing/shotgun/prespawned(src)
+		new /obj/item/storage/pouch/ammo(src)
+
+// Side-arms Kits
+/obj/item/storage/box/bs_kit/stallion
+	name = "\improper Stallion Secondary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/colt/ten/dark(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/ammo_magazine/magnum_40(src)
+		new /obj/item/clothing/accessory/holster/leg(src)
+
+/obj/item/storage/box/bs_kit/lamia
+	name = "\improper Lamia Secondary Kit"
+	desc = "The standard Blackshield equipment kit."
+
+	populate_contents()
+		new /obj/item/gun/projectile/lamia/dark(src)
+		new /obj/item/ammo_magazine/kurtz_50/lethal(src)
+		new /obj/item/ammo_magazine/kurtz_50/lethal(src)
+		new /obj/item/clothing/accessory/holster/leg(src)
+
+/obj/item/storage/box/bs_kit/martin
+	name = "\improper Martin Secondary Kit"
+	desc = "A kit designed for energy-based weaponry."
+
+	populate_contents()
+		new /obj/item/gun/energy/gun/martin(src)
+		new /obj/item/cell/small/high(src)
+		new /obj/item/cell/small/high(src)
+		new /obj/item/clothing/accessory/holster/leg(src)
+
+// Armor Kits
+/obj/item/storage/box/bs_kit/standard_armor
+	name = "\improper Standard Armor Kit"
+	desc = "An armor kit."
+
+	populate_contents()
+		new /obj/item/clothing/suit/armor/platecarrier/militia(src)
+		new /obj/item/clothing/head/helmet/ballistic/militia(src)
+
+/obj/item/storage/box/bs_kit/flak_armor
+	name = "\improper Flak Armor Kit"
+	desc = "An armor kit."
+
+	populate_contents()
+		new /obj/item/clothing/suit/armor/flackvest/militia(src)
+		new /obj/item/clothing/head/helmet/ballistic/militia(src)
+
+/obj/item/storage/box/bs_kit/laser_armor
+	name = "\improper Ablative Armor Kit"
+	desc = "An armor kit."
+
+	populate_contents()
+		new /obj/item/clothing/suit/armor/laserproof(src)
```

Comment:
this is the mirror armor not aborbing meaning this is the best laser armor your can get in the game other then SI's verson. It sholdnt be that good

Reply:
Which variant should I use then?